docs/src/cross_building.adoc

```diff
@@ -0,0 +1,190 @@
+include::_{backend}_settings.adoc[]
+
+= Cross Building
+
+== Setting up a cross build environment
+
+=== Requirements
+
+Setting up a cross build development environment requires a suitable host environment to
+install and run all tooling. Currently TAOX11 (AXCIOMA) only supports mainstream Linux desktop
+environments (OpenSuSE, Debian, Fedora, CentOS, RHEL) as host environment for cross building.
+
+All development tool prerequisites that apply for building TAOX11 (AXCIOMA) for the host platform
+itself also apply for cross building.
+In addition a cross build toolchain for the intended target environment needs to be installed. Typically
+such toolchain packages include host compiler, linker, librarian etc. binaries as well as a target system
+root (sysroot) SDK providing target versions of libraries etc.
+
+=== Installing
+
+Installing the TAOX11 (AXCIOMA) source code for the cross build is identical to the process for a host
+build. Simply use Git to clone the base axcioma.git repository and use the `bin/brix11 bootstrap` command
```

Comment:
git

Reply:
The product is "Git" the tool is "git". I am referring to the product here.